rfcs/2020-v3-telemetry.md

```diff
@@ -0,0 +1,203 @@
+<!-- Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+- Start Date: 2020-03-24
+- RFC PR: (leave this empty, to be filled in later)
+- Authors: Ross Pfahler
+
+# React Spectrum v3 Component Telemetry
```

Comment:
I had a weird thought, do we need this to be part of the compiled app where it reports on how much usage a component gets or just something that reports when react-spectrum is included in a production build? I know including it in production compiled and deployed code makes things like identifying performance issues easier.